cypress/support/fragments/data_import/mapping_profiles/fieldMappingProfiles.js

```diff
@@ -12,34 +11,21 @@ const openNewMappingProfileForm = () => {
 };
 
 const closeViewModeForMappingProfile = () => {
-  cy.do(Button('Close'));
-};
-
-// TODO create search mapping profile instead wait
-const waitLoadingMappingProfile = () => {
-  cy.get('[id="full-screen-view-content"]', getLongDelay())
-    .should('be.visible');
-  cy.expect(actionsButton.exists());
-};
-
-const waitLoadingList = () => {
-  cy.get('[id="mapping-profiles-list"]', getLongDelay())
-    .should('be.visible');
-  cy.expect(actionsButton.exists());
+  cy.get('[aria-label="Close "]').click();
```

Comment:
Can you use interactor like `Button` istead of `cy.get`?

Reply:
Thanks! I use interactor